src/main/java/shop/menu/admin/UsersMenu.java

```diff
@@ -0,0 +1,82 @@
+package shop.menu.admin;
+
+import shop.menu.CredentialsMenu;
+import shop.menu.Menu;
+import shop.service.AdminService;
+import shop.service.impl.AdminServiceImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class UsersMenu implements Menu {
+    private CredentialsMenu credentialsMenu = new CredentialsMenu();
+    private AdminService adminService = new AdminServiceImpl();
+    private Scanner scanner = new Scanner(System.in);
+    private List<String> options = new ArrayList<>();
+
+    @Override
+    public void addOptions() {
+        options.add("1. Block user");
+        options.add("2. Unblock user");
+        options.add("3. Delete user");
+        options.add("0. Go back");
+    }
+
+    @Override
+    public void show() {
+        addOptions();
+        showOptions(options);
+
+        while (true) {
+            try {
+                int choice = scanner.nextInt();
+                switch (choice) {
+                    case 1:
+                        System.out.printf("Blocked Users: \n%s%n",
+                                adminService.getActiveUsers());
+                        if (adminService.blockUser(credentialsMenu.getEmail())) {
```

Comment:
there is no need to make it boolean and return state. If something went wrong, method throw exception and you won`t go through flow to sout line in menu. So you can just leave
blockUser()
sout("User has blocked"))